api/sheets/get_variable.md

```diff
@@ -13,28 +13,63 @@ grand_parent: API
 Given a list of [Place](https://browser.datacommons.org/kg?dcid=Place) DCIDs, a [StatisticalVariable](https://browser.datacommons.org/kg?dcid=StatisticalVariable), and a date, get the measurements of that variable at those places on that date.
 
 **Arguments**
-*   `dcids` - place DCIDs
+*   `dcids` - place DCIDs as a single value, a row, or a column
 *   `variable` - The StatisticalVariable to get
-*   `date` - The date to observe, e.g. 2017, "2017", "2017-12"
+*   `date` - The dates to observe, e.g. 2017, "2017", "2017-12". This can be a single value, a row, or a column.
```

Comment:
Can maybe explain this is ISO 8601 format. For a split second I was thinking 2017-2012.

Reply:
Nit: should dcid and date arguments have the same plurality?

Reply:
I specified that it is ISO 8601 format. I should change "date" to "dates", but that requires changing the app. I filed b/157822238 for that.

Reply:
Also, the function can take a Google Sheets date object instead of a string, so I specified that.

Reply:
Also, the function can take a Google Sheets date object instead of a string, so I specified that.